selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/WorkspaceDetailsSingleMachineTest.java

```diff
@@ -133,12 +132,6 @@ public void checkOverviewTab() {
   public void checkWorkingWithInstallers() {
     workspaceDetails.selectTabInWorkspaceMenu(INSTALLERS);
 
-    // check both versions of the 'Workspace API' installer
```

Comment:
IMHO it's better to check version by regex `\d+\.\d+\.\d+` than don't have verification of appearance of "Workspace API" installers at all.